src/server/components/Medlemskapsgrunnlag.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { IMedlemskap } from '../../typer/dokumentApi';
+
+interface Props {
+  medlemskap: IMedlemskap;
+}
+
+const Medlemskapsgrunnlag: React.FC<Props> = ({ medlemskap }) => {
+  return (
+    <>
+      <h3>Grunnlag</h3>
+      <div>Personstatus: {medlemskap.registergrunnlag.folkeregisterpersonstatus}</div>
+      <div>Statsborgerskap: {medlemskap.registergrunnlag.nåværendeStatsborgerskap.join(', ')}</div>
+      <div>
+        Tidligere statsborgerskap:{' '}
+        {medlemskap.registergrunnlag.statsborgerskap.length === 0
+          ? 'Ingen data'
+          : medlemskap.registergrunnlag.statsborgerskap.map((statsborgerskap, i) => {
+              return (
+                <div key={i}>
+                  Fra: {statsborgerskap.gyldigFraOgMedDato || 'Ikke angitt'} | Til:{' '}
+                  {statsborgerskap.gyldigTilOgMedDato || 'Ikke angitt'} | Land:{' '}
+                  {statsborgerskap.land}
+                </div>
+              );
+            })}
+      </div>
```

Comment:
Kunne man brukt `<p/>` tags for å vise tekster i stedet for alle disse `<div>`-ene? Det blir lettere å lese og man får proper semantics. 

Litt usikker på hvor mye det har å si siden et ser ut som at dere lager en pdf - gjør dere det? Eller vises det bare som en pdf? 🤔

Reply:
Vi bør bytte ut div-ene med noe mer meningsfullt, ja :) Kanskje til og med listevisning for enkelte ting. I første omgang er det bare en POC for å se hvor lett/vanskelig det er å generere en pdf med litt html og data fra ef-sak.